lib/extensions/ar_private_attribute.rb

```diff
@@ -0,0 +1,22 @@
+module ArPrivateAttribute
+  extend ActiveSupport::Concern
+
+  included do
+    class_attribute :private_attribute_names, :default => []
```

Comment:
If a subclass marks an attribute as private, then what will happen?  I feel like it should mark it private for the entire class chain.

Or even better, maybe we shouldn't even allow marking attributes as private except where that attribute is defined - I'm not sure how to implement that though.

Reply:
this implementation has it just show private for the current class - not child classes.

so you are thinking that `visible_attributes` will be recursive across all the children?

Reply:
thanks for the question - added tests to ensure that it is hidden for all classes and child classes.

couldn't figure out how to ensure that child classes also hid parent classes.